backend/dissemination/forms.py

```diff
@@ -2,10 +2,16 @@
 
 
 class SearchForm(forms.Form):
+    choices_YoN = (("True", "Yes"), ("False", "No"))
```

Comment:
does this need to be here? I don't see it referenced anywhere in the `dissemination app`

Reply:
Oops, that's a leftover copy/paste. Good catch!